frontend/src/containers/Collection/CollectionManage/CollectionManage.js

```diff
@@ -154,14 +152,33 @@ class CollectionManage extends Component {
                           showWarningContentText={false}
                         />
                     </div>
+                    <hr color="gray" />
+                    <div className="ManageButtonItems">
+                        <div className="ManageButtonText" id="transferOwnershipText">
+                            <h5>
+                                Transfer the ownership of this collection
+                                to the other member of this collection
+                            </h5>
+                            <div className="warning-block">
+                                <h5 id="warning-text">WARNING:&nbsp;</h5>
+                                <h5 id="warning-content">This action cannot be undone</h5>
+                            </div>
+                        </div>
+                        <TransferOwnershipModal history={this.props.history} />
+                    </div>
+                    <hr color="gray" />
                     <div className="ManageButtonItems">
                         <div className="ManageButtonText" id="deleteCollectionText">
-                            <h5>Delete this collection.</h5>
-                            <h5>WARNING: This action cannot be undone.</h5>
+                            <h5>Delete this collection</h5>
+                            <div className="warning-block">
+                                <h5 id="warning-text">WARNING:&nbsp;</h5>
+                                <h5 id="warning-content">This action cannot be undone</h5>
+                            </div>
                         </div>
                         <WarningModal
                           id="delete-warningmodal"
-                          openButtonText="Delete this collection"
+                          variant="danger"
+                          openButtonText="Delete Collection"
                           whatToWarnText={`Delete colelction: ${this.props.selectedCollection.title}`}
```

Comment:
Minor: Could you change colelction to collection?

Reply:
or I could merge it and change it in my next pr

Reply:
Thank you! I'll fix it now.